src/main/java/org/jitsi/videobridge/health/Health.java

```diff
@@ -192,7 +161,7 @@ protected void performCheck()
             throw new Exception("Failed to bind single-port");
         }
 
-        checkXmppConnection(videobridge);
+        // TODO: check if ClientConnectionImpl is configured and connected.
```

Comment:
Do we need to address this before merging?

Reply:
No, the behavior doesn't change. We were only checking the component connection anyway.